source/memory.tex

```diff
@@ -6053,6 +6056,18 @@
 Nothing unless the constructor for \tcode{T} throws.
 \end{itemdescr}
 
+\indexlibrarymember{destroy}{polymorphic_allocator}%
+\begin{itemdecl}
+template<class T>
+  void destroy(T* p);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to \tcode{p->\~T()}.
```

Comment:
Is it intended to introduce SFINAE constraints implied by "Equivalent to" here ([[structure.specifications]/4](https://eel.is/c++draft/structure.specifications#4))? This seemingly makes the changes more than pure undeprecation.
I've attempted to file an LWG issues for this.

Reply:
This does not imply SFINAE. The use of "_Effects_: Equivalent to" means that other [structure.specifications] elements such as _Constraints_ and _Throws_ are inherited, but the delete-expression is not a standard library component and is not described by any specification using _Constraints_, _Throws_ etc.

If it said "expression-equivalent to" that would imply SFINAE (and conditional noexcept etc).